src/main/kotlin/app/revanced/patches/youtube/layout/spoofappversion/SpoofAppVersionPatch.kt

```diff
@@ -17,8 +17,8 @@ import com.android.tools.smali.dexlib2.iface.instruction.OneRegisterInstruction
 
 @Patch(
     name = "Spoof app version",
-    description = "Tricks YouTube into thinking you are running an older version of the app. " +
-            "One of the side effects also includes restoring the old UI.",
+    description = "Tricks YouTube into thinking you are running a different version of the app. " +
+            "This can be used to restore the old UI.",
```

Comment:
In LisoUseInAlKyrios' review, they suggested `This can be used to restore **old features and UI layouts**.`

To merge it with your suggestion, I'm thinking of doing something like: `This can be used to restore some old UI elements and features.`

Any thoughts on this?

Reply:
```suggestion
            "This can be used to restore some old UI elements and features.",
```